scripts/talos/getBranchName/index.js

```diff
@@ -0,0 +1,11 @@
+const getDate = () => {
+  const today = new Date();
+  const date = `${today.getFullYear()}${today.getMonth() +
+    1}${today.getDate()}`;
+  const time = `${today.getHours()}${today.getMinutes()}${today.getSeconds()}`;
+  return date + time;
+};
+
+const getBranchName = () => `Talos/${getDate()}`;
```

Comment:
How confident are we that two instances of `talos` can never be running at the same second and therefore both interact on the same branch?